### PR TITLE
Global replace of integral single-precision literals with integer literals.

### PR DESCRIPTION
**Pull request by romoore**
Global replace of integral single-precision literals with integer literals.

Regular expression used:
```perl
/[^\w]([0-9]+)\.[0]*[fF]/$1/
```

### CODE REVIEW

File: components/List/examples/supplemental/CollectionViewListCell.m [hunk 1]
```diff
@@ -17,20 +17,16 @@
...
+static const CGFloat kImageHeight = 40;
+static const CGFloat kWithImageRightPadding = 12;
+static const CGFloat kTextVerticalPadding = 12;
 static const CGFloat kTitleColorOpacity = 0.87f;
 static const CGFloat kDetailsColorOpacity = 0.6f;
 
-static inline UIFont *defaultTitleFont(void) {
-  return [UIFont systemFontOfSize:16.f];
-}
+static inline UIFont *defaultTitleFont(void) { return [UIFont systemFontOfSize:16]; }
```
[the remark this answers is not in the document]
Comment (romoore) [reply components/List/examples/supplemental/CollectionViewListCell.m:27]: You're wrong, clang-format! This was formatted by google's internal formatter. ;-)

File: components/List/examples/supplemental/CollectionViewListCell.m [hunk 2]
```diff
@@ -17,20 +17,16 @@
...
 
-static inline UIFont *defaultDetailsFont(void) {
-  return [UIFont systemFontOfSize:14.f];
-}
+static inline UIFont *defaultDetailsFont(void) { return [UIFont systemFontOfSize:14]; }
```
[the remark this answers is not in the document]
Comment (romoore) [reply components/List/examples/supplemental/CollectionViewListCell.m:29]: Same. Google's internal formatting tools disagree, and I'm pretty sure they win. :D

File: components/private/Math/src/MDCMath.h [hunk 3]
```diff
@@ -93,9 +93,7 @@ static inline CGFloat MDCHypot(CGFloat x, CGFloat y) {
 }
 
 // Checks whether the provided floating point number is exactly zero.
-static inline BOOL MDCCGFloatIsExactlyZero(CGFloat value) {
-  return (value == 0.f);
-}
+static inline BOOL MDCCGFloatIsExactlyZero(CGFloat value) { return (value == 0); }
```
[the remark this answers is not in the document]
Comment (romoore) [reply components/private/Math/src/MDCMath.h:96]: Sorry, clang-format. You lose and Google wins. ;-)

[the remark this answers is not in the document]
Comment (jverkoey) [reply components/private/Math/src/MDCMath.h:96]: Looks like clang-format is out-of-date. I'll generate a new build.

**Comment by jverkoey** [comment]
Please add the regular expression you used to perform this cleanup to the PR description.

**Comment by romoore** [comment]
@jverkoey Done.

**Review by jverkoey — APPROVED** [review]
LGTM but due to size and potential for behavioral changes let's mark as risky.

**Pull request merged**